### DISCUSSION TITLE
How to display symbols/icons in starship.toml file in VS Code?

### DISCUSSION THREAD

**Discussion by bluelephant825** [starship/starship#4409] [Q&A] [answered]
Hello,

I've installed starship via brew on macOS Monterey 12.6. I'd like to edit the config file starship.toml in VS Code 1.71.2 (Universal), but I'm having difficulties showing the symbols/icons as can be seen in an extract below:

<img width="825" alt="Screenshot 2022-09-26 at 19 15 43" src="https://user-images.githubusercontent.com/4818315/192339710-3f5fa01f-3d90-4346-9bfd-ab9f27107f7a.png">

My settings.json file for VS Code is as follows:

```
{
  "editor.fontLigatures": "'calt', 'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'zero', 'onum'",
  "workbench.iconTheme": "material-icon-theme",
  "editor.fontFamily": "'Source Code Pro', 'Fira Code', 'Cascadia Code', 'Courier New', 'Noto Emoji', monospace",
  "editor.formatOnSave": true,
  "editor.wordWrap": "on",
  "editor.accessibilitySupport": "off",
  "editor.fontSize": 15,
  "terminal.integrated.env.osx": {
      "FIG_NEW_SESSION": "1"
  },
  "security.workspace.trust.untrustedFiles": "open",
  "terminal.integrated.inheritEnv": false,
  "editor.guides.bracketPairs": true,
  "editor.guides.bracketPairsHorizontal": true,
  "workbench.editor.enablePreview": false,
  "terminal.integrated.localEchoStyle": "dim",
  "terminal.integrated.fontFamily": "MesloLGS NF",
  "terminal.integrated.fontWeight": "normal",
  "terminal.external.osxExec": "iTerm.app",
  "redhat.telemetry.enabled": true
}
```

I'm not sure what changes need to be made for the proper symbols/icons to appear in the VS Code editor.

**Comment by davidkna** [comment, marked as answer]
Are the icons nerd font icons? Please try adding [a nerd font](https://www.nerdfonts.com/) to the font list.

**Reply by bluelephant825** [reply]
I tried different nerd fonts from [this page](https://www.nerdfonts.com/font-downloads). Cascadia Code, FiraCode and Source Code Pro all failed. Finally, MesloLGS NF worked. It's really a bit of a random hit or miss it seems!

**Reply by davidkna** [reply]
@odebroqueville Maybe you weren't using the correct font names? They all have a " Nerd Font"-suffix, and some fonts like "Cascadia Code" use a different name.

**Reply by bluelephant825** [reply]
@davidkna You're absolutely right! I downloaded & re-installed all the fonts and changed the VS Code Editor Font Family setting to "'SauceCodePro Nerd Font', 'CaskaydiaCove Nerd Font', 'FiraCode Nerd Font', 'Courier New', monospace" and each nerd font works perfectly now! Thank you so much!
